### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by landanhu**

### CODE REVIEW

File: docs/getting-started/transaction.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ All these types should also contain metadata, such as [network ID](./chain-id-an
 | Tx type              | Inputs                 | Outputs                                                                      | Fees involved? | Obfuscated? |
 | -------------------- | ---------------------- | ---------------------------------------------------------------------------- | -------------- | ----------- |
 | `TransferTx`         | UTXOs                  | UTXOs                                                                        | Yes            | Yes         |
-| `DepositStakeTx`     | UTXOs                  | Depostit to specified account’s `bonded` amount                              | Yes            | Yes         |
+| `DepositStakeTx`     | UTXOs                  | Deposit to specified account a `bonded` amount                              | Yes            | Yes         |
```
Comment (lezzokafka) [inline docs/getting-started/transaction.md]:
```suggestion
| `DepositStakeTx`     | UTXOs                  | Deposit to the `bonded` amount of the specified account                              | Yes            | Yes         |
```

**Review by lezzokafka — COMMENTED** [review]
Nice catches :), just left one suggestion.

**Comment by landanhu** [comment]
> Nice catches :), just left one suggestion.

Deposit the `bonded` amount to the specified account ?  @lezzokafka

**Comment by lezzokafka** [comment]
> Deposit the `bonded` amount to the specified account ? @lezzokafka

Thought about what should be the correct way to phrase this: 
_Spent the UTXO (the inputs) and "credit" the `bonded` amount balance_ 

I was trying to do it in a "Deposit to the credit of ....." way, might be a bit clumsy tho.

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**